### ISSUE TITLE
disableStatCache

### ISSUE THREAD

**Issue by ariselseng** [thephpleague/flysystem-sftp#62]
Hi,
Could we get an option to disable the stat cache from phpseclib?
I need to know the size of a file without any caching.
See other issue here: https://github.com/thephpleague/flysystem/issues/949#issuecomment-403291667

I tested this locally and it worked:
```php
    /**
     * Connect.
     */
    public function connect()
    {
        $SFTP = new SFTP($this->host, $this->port, $this->timeout);
        $SFTP->disableStatCache();
        $this->connection = $this->connection ?: $SFTP;
        $this->login();
        $this->setConnectionRoot();
    }

**Comment by frankdejonge** [comment]
Fixed in `1.0.16`

**Comment by ariselseng** [comment]
@frankdejonge Wow that was fast. Thanks!
